aws/resource_aws_ssm_maintenance_window_target.go

```diff
@@ -17,6 +17,17 @@ func resourceAwsSsmMaintenanceWindowTarget() *schema.Resource {
 		Read:   resourceAwsSsmMaintenanceWindowTargetRead,
 		Update: resourceAwsSsmMaintenanceWindowTargetUpdate,
 		Delete: resourceAwsSsmMaintenanceWindowTargetDelete,
+		Importer: &schema.ResourceImporter{
+			State: func(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+				idParts := strings.Split(d.Id(), ":")
+				if len(idParts) != 2 || idParts[0] == "" || idParts[1] == "" {
+					return nil, fmt.Errorf("Unexpected format of ID (%q), expected WINDOW_TARGET_ID:WINDOW_ID", d.Id())
```

Comment:
Let's update this to look like [`aws_ssm_maintenance_window_task`'s import](https://www.terraform.io/docs/providers/aws/r/ssm_maintenance_window_task.html#import) ( WINDOW_ID/WINDOW_TARGET_ID) so operators have a similar experience between these two resources. 😄 

For import IDs where one resource is containing another, we do try to put container, delimiter, contained. 👍